### PR TITLE
fix broken links for test-runner-interface and test-runner-docker

### PR DESCRIPTION
**Pull request by peterschussheim**
Updated documentation links for test-runner-interface and test-runner-docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ Catch can report the tests results in [JUnit][junit] formatted xml when enabled,
 
 This file is parsed with Python and the [junitparser][junitparser-lib] library in the `process.py` script that outputs a `results.json` file that respects the test runners specifications.
 
-[test-runner-interface]: https://github.com/exercism/automated-tests/blob/master/docs/interface.md
-[test-runner-docker]: https://github.com/exercism/automated-tests/blob/master/docs/docker.md
+[test-runner-interface]: https://github.com/exercism/docs/blob/main/building/tooling/test-runners/interface.md
```
Comment (ErikSchierboom) [inline README.md:33]:
```suggestion
[test-runner-interface]: https://exercism.org/docs/building/tooling/test-runners/interface
```

File: README.md [hunk 2]
```diff
@@ -30,8 +30,8 @@ Catch can report the tests results in [JUnit][junit] formatted xml when enabled,
...
+[test-runner-docker]: https://github.com/exercism/docs/blob/main/building/tooling/docker.md
```
Comment (ErikSchierboom) [inline README.md:34]:
```suggestion
[test-runner-docker]: https://exercism.org/docs/building/tooling/test-runners/docker
```

**Review by ErikSchierboom — COMMENTED** [review]
In https://github.com/exercism/cpp-test-runner/issues/36#issuecomment-915048624 I suggested two small changes to the links, where instead of linking to the docs repo, we link to the docs part of the website. I've suggested the changes.

File: README.md (hunk 2 above)
Comment (peterschussheim) [reply README.md]: Thanks for these suggestions Erik, I've pushed a new commit with the changes.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**